examples/cutout_export_request.py

```diff
@@ -0,0 +1,73 @@
+"""
+===================================
+Exporting data with cutout requests
+===================================
+
+This example shows how to submit a data export request with a cutout request using the ``im_patch`` command.
+"""
+
+import os
+
+import drms
+
+###############################################################################
+# Create DRMS client, uses the JSOC baseurl by default, set debug=True to see the DRMS query URLs.
+
+client = drms.Client(verbose=True)
+
+# This example requires a registered export email address. You can register
+# JSOC exports at: http://jsoc.stanford.edu/ajax/register_email.html
+email = 'nabil.freij@gmail.com'
+
+# Download directory
+out_dir = os.path.join('downloads')
```

Comment:
pathlib?

Reply:
All the examples will need updating.

Reply:
Yeah I would just hold off on updating all the gallery examples until later